### ISSUE TITLE
toolbar doesn't collapse responsively on some sites.

### ISSUE THREAD

**Issue by SpiritLevel** [silverstripe/silverstripe-globaltoolbar#40]
Toolbar doesn't collapse on small devices for the following sites:

userhelp.silverstipe.org
docs.silverstripe.org
api.silverstripe.org

All other sites linked from within toolbar appear to have a properly collapsing toolbar.

**Comment by tractorcow** [comment]
Would it be a simple case of updating and re-deploying the toolbar?

I double checked and you're right about it working on some sites but not the above. ;)

**Comment by SpiritLevel** [comment]
userhelp and docs uses ^4.0 so that should grab the latest toolbars version 4.1.1, right? Or do you mean something else by update and redeploy ? :)

**Comment by tractorcow** [comment]
Yeah but it might not have been updated in a while. :) When was the last deployment to userhelp or docs? @camfindlay?

**Comment by camfindlay** [comment]
Userhelp was in the last few weeks, dev docs was last deployed end of last year.

**Comment by camfindlay** [comment]
I think the issue is the way the toolbar gets included in the subdomain sites which leads to it not being responsive. Feel free to tinker and see.

**Comment by mandrew** [comment]
Hi @SpiritLevel, one issue is that both the docs site and silverstripe.org were built with different versions of bootstrap (unfortunately) so there will be issues with getting both of them to talk nicely. The second issue will be with the information that comes from the silverstripe.org menu and is displayed in the responsive menu. A bit of work will need to take place to get this working the same but feel free to give it a go and I'd be happy to review the work.

**Comment by SpiritLevel** [comment]
In the interests of easy maintenance, it seems to me that the same css/js framework version and code should be used on all these ss.org sites :P What's involved in unifying all these things ? I am digging into addons to update the search and considering how to make it responsive so perhaps it is a good idea to plan this out now ? What do you all think ?
